docs/metrics.md

```diff
@@ -17,8 +17,7 @@ Measurement accuracy can be impacted by other parallel processing that might be
 ## Change log
 This list is to help notify if there are additions, changes or removals to metrics.
 
-- Add libovsdb metrics - ovnkube_master_update_messages_total, ovnkube_master_table_updates_total, ovnkube_master_disconnects_total,
-  ovnkube_master_monitors.
+- Add libovsdb metrics - ovnkube_master_disconnects_total and ovnkube_master_monitors.
```

Comment:
I just realized that `ovnkube_master_disconnects_total` doesn't tell you anything about what kind of disconnect it measures, maybe add libovsdb to the name?

Reply:
True. The tooltip or description in OCP console will give context but the name isnt good.

Ill prefix it with "database", so itll be ovnkube_master_database_disconnects_total. 
Ill do the same to monitors.
I originally didnt want to rename the metrics in case anyone was using it but since they were broken anyway, why not :)

Reply:
Ill see what dan says in libovsdb.

Reply:
or maybe ovsdb - the reason I am not picking libovsdb is the default metric "namespace" is called "libovsdb" but for ovn-k we are overriding it with "ovnkube".

Reply:
I think new name makes perfect sense